Source/Scene/ModelInstanceCollection.js

```diff
@@ -302,17 +304,44 @@ define([
             vertexShaderCached = instancedSource;
 
             if (usesBatchTable) {
-                instancedSource = collection._batchTable.getVertexShaderCallback()(instancedSource);
+                instancedSource = collection._batchTable.getVertexShaderCallback(true)(instancedSource);
             }
 
             return instancedSource;
         };
     }
 
-    function getFragmentShaderCallback(collection) {
+    // TODO : duplicate of Batched3DModel3DTileContent
```

Comment:
I guess it should go into its own `@private` file unless there is a util 3D Tiles or glTF file.